src/main/java/org/traccar/protocol/AquilaProtocolDecoder.java

```diff
@@ -121,6 +121,18 @@ public AquilaProtocolDecoder(Protocol protocol) {
             .number("(?:[01],){4}")              // reserved
             .number("(d+),")                     // external voltage
             .number("(d+),")                     // internal voltage
+            .or()                                   // EXTENDED for OBD Devices
+            .number("(d+),")                     // course
+            .number("(d+),")                     // satellites
+            .number("(d+.d+),")                  // hdop
+            .number("(?:d+,){2}")                // reserved - skip
+            .number("(d+),")                     // voltage of analog I/P - skip in code
+            .number("(?:d+),")                   // event flag - skip
+            .number("(d+),")                     // external voltage
+            .number("(d+),")                     // internal voltage
+            .number("(?:d+),")                   // trip time - skip
```

Comment:
If we don't capture something, please remove the capture group for it. Same in other places.